docs/source/02-Analytics-Query/draft/Success-Failed-Event-Count-By-Event-Name.sql

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
the title of this query is a bit confusing- since it's by event count you could just put count by event name.